src/msg/async/AsyncConnection.cc

```diff
@@ -443,7 +449,18 @@ void AsyncConnection::process() {
     case STATE_ACCEPTING: {
       center->create_file_event(cs.fd(), EVENT_READABLE, read_handler);
       state = STATE_CONNECTION_ESTABLISHED;
-
+      if (async_msgr->cct->_conf->mon_min_delay_socket) {
+        if (async_msgr->get_mytype() == CEPH_ENTITY_TYPE_MON
+            && peer_is_mon()) {
+          ssize_t r = cs.set_priority(cs.fd(), SOCKET_PRIORITY_MIN_DELAY, target_addr.get_family());
+          if (r < 0) {
+            ldout(async_msgr->cct, 0) << __func__ << " couldn't set SO_PRIORITY to "
+                 << SOCKET_PRIORITY_MIN_DELAY << ": " << cpp_strerror(r) << dendl;
+            protocol->fault();
+            return;
+          }
```

Comment:
Given that `NetHandler::set_priority()` swallows all errors, this looks like a dead branch.

Reply:
returning error and retry  may be better than swallowing all errors, i will change cs.set_priority code to return error .

Reply:
already remove this branch